cdatransform/extract/pdc.py

```diff
@@ -123,6 +217,26 @@ def get_file_metadata(file_metadata_record) -> dict:
         ]
     }
 
+        
+def agg_cases_info_for_study(study,demo,diag,samp,added_info):
```

Comment:
can we call this sample rather than samp.

Reply:
Good idea. Will alter and push